TRANSLATION.md

```diff
@@ -12,20 +12,20 @@ Suggestion on words and terms:
 
 | Original word/term | Suggestion |
 | ------------------ | ---------- |
-| arrow function | *arrow function* |
+| arrow function | función flecha |
 | bug | error |
 | Bundler | *Bundler* |
 | camelCase | *camelCase* |
 | callback | *callback* |
 | controlled component | *controlled component* |
 | debugging | depuración |
 | library | biblioteca |
-| lowercase | *lowercase* |
+| lowercase | minúscula(s) |
 | props | *props* |
 | React element | Elemento de React |
-| state | *state* |
-| string | *string* |
-| template literals | *template literals* |
+| state | estado |
+| string | cadena de texto |
+| template literals | plantilla de cadena (de texto) |
```

Comment:
This sounds so weird in Spanish 😅

Reply:
Because of the same reasons as https://github.com/reactjs/es.reactjs.org/pull/23#discussion_r253056553, I think we should leave `template literals` as is, but really I don't have strong opinions about it though.

Reply:
I've seen "string" defined as `cadena de texto` or `cadena de caracteres`, but I've never seen it used like that. 
Same thing goes for "template literals".

For me those are terms that encapsulate a concept and although they can be translated, the meaning just isn't the same.

Reply:
I also think we should leave `template literals` as is, also `string`.

Reply:
I would incline towards `cadena de caracteres` for `string`. 

For `template literals`, I think `plantillas de texto` does a great job, though it still requires some consensus behind it to fully convey the meaning.